### PR TITLE
[CET-9460] Backstage version update from 1.25.2 to 1.27.7

### PR DESCRIPTION
**Pull request by PavelZurek**
## Change description

[CET-9460] Backstage version update from 1.25.2 to 1.26.5

Used command `yarn backstage-cli versions:bump`

## Type of change

- [ ] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)
- [x] Improvement (improves codebase without changing functionality)

### Development

- [ ] The changelog has been updated as appropriate
- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

[CET-9460]: https://cortex1.atlassian.net/browse/CET-9460?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 2.10.0
```
[the remark this answers is not in the document]
Comment (PavelZurek) [reply CHANGELOG.md:3]: That's ready to be merged after #169

File: backstage.json [hunk 2]
```diff
@@ -1,3 +1,3 @@
 {
-  "version": "1.25.2"
+  "version": "1.26.5"
```
Comment (cristinabuenahora) [inline backstage.json:2]: Can we upgrade to 1.27.6

Comment (PavelZurek) [reply backstage.json]: I used latest - 1.27.7, hope it's okay

File: CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (FoodProduct) [reply CHANGELOG.md:3]: Are we sure we still want that sequencing? 🙂

[the remark this answers is not in the document]
Comment (PavelZurek) [reply CHANGELOG.md:3]: That's a question for @cristinabuenahora I think.

**Review by cristinabuenahora — DISMISSED** [review]

**Review by cristinabuenahora — APPROVED** [review]

**Comment by alexef** [comment]
Can we do the same in the backend plugin? https://github.com/cortexapps/backstage-backend-plugin/pull/52

**Pull request merged**